src/yaws_config.erl

```diff
@@ -3620,8 +3621,10 @@ str2term(Str0) ->
 
 check_ciphers([], _) ->
     ok;
-check_ciphers([Spec|Specs], L) ->
-    case lists:member(Spec, L) of
+check_ciphers([{Kex, Cipher, Mac, Prf} = Spec|Specs], L) ->
+    CipherMap = maps:from_list([{key_exchange, Kex}, {cipher, Cipher},
+                                {mac, Mac}, {prf, Prf}]),
+    case lists:member(CipherMap, L) of
```

Comment:
hm...is Maps sorted? else are you sure an "identical" Map in L will match?

Reply:
Maps with the same content will match.
```
4> #{a => 1, b => 2} == #{b => 2, a => 1}.
true
5> #{a => 1, b => 2} =:= #{b => 2, a => 1}.
true